pkg/nvlib/device/device.go

```diff
@@ -208,6 +209,33 @@ func (d *device) IsMigEnabled() (bool, error) {
 	return (mode == nvml.DEVICE_MIG_ENABLE), nil
 }
 
+// HasGpuFabricInfo() checks if a device has IMEX capabilities.
+func (d *device) HasGpuFabricInfo() (bool, error) {
+	if d.lib.hasSymbol("nvmlDeviceGetGpuFabricInfo") {
+		_, ret := d.GetGpuFabricInfo()
+		if ret == nvml.ERROR_NOT_SUPPORTED {
+			return false, nil
+		}
```

Comment:
Do we know if this is a valid error from this API call?

Reply:
https://github.com/NVIDIA/go-nvml/blob/main/pkg/nvml/const.go#L1159